snf/inject.go

```diff
@@ -254,15 +257,16 @@ func (s *Sender) SendBulk(pkts [][]byte) error {
 		return err
 	}
 
+	s.guardPkts = pkts
 	s.pkts = s.pkts[:0]
 	s.len = s.len[:0]
 	for _, pkt := range pkts {
-		s.pkts = append(s.pkts, pkt...)
+		s.pkts = append(s.pkts, uintptr(unsafe.Pointer(&pkt[0])))
```

Comment:
`&pkts[i]`?

Reply:
My bad, sorry

Reply:
`s.pkts` has pointers to byte arrays, which are raw packets. `&pkts[i]` is a pointer to a slice that has a pointer to a byte array, which can be obtained by slicing, so in this case, it should be `&pkts[i][0]` equivalent to `&pkt[0]`.